src/public/parts/shells/chat/public/hub/messages.css

```
@@ -1,10 +1,18 @@
 /* Hub 消息区：布局由 DaisyUI chat 组件承担，此处仅保留无法用工具类表达的规则 */
 
 .messages {
+	--chat-gap: 0.375rem;
+	/* 与 avatar_block `w-10` 对齐；改尺寸只动这一处 */
+	--message-avatar-size: 2.5rem;
 	overflow-y: auto;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)